src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2580,3 +2580,5 @@ Finished=Finished
 Finished\ writing\ metadata\ for\ library\ %0\ (%1\ succeeded,\ %2\ skipped,\ %3\ errors).=Finished writing metadata for library %0 (%1 succeeded, %2 skipped, %3 errors).
 Processing...=Processing...
 Writing\ metadata\ to\ {}=Writing metadata to {}
+
+Get\ More\ Themes= Get More Themes!
```

Comment:
Here you need to remove the ! as well from the right side, both sides have to have the same text